dynamic_programming/longest_palindromic_subsequence.py

```diff
@@ -0,0 +1,56 @@
+"""
+Longest Palindromic Subsequence Problem:
+Find the longest sequence of characters from a given input string that
+is the same backwards and forwards
+https://en.wikipedia.org/wiki/Longest_common_subsequence_problem
+"""
+
+
+def longest_palindromic_subsequence(s: str, length: int) -> int:
+
+    """
+    Computes the longest Palindromic sequence of a string
+
+    :param s: str, the string we provide
+    :param length: int, the length of the string
+    :return L[0][n - 1]: int, the length of the longest Palindromic subsequence
+
+
+
+    >>> longest_palindromic_subsequence("ABBCDABBC",9)
+    5
+    >>> longest_palindromic_subsequence("ABACCG",6)
+    3
+    >>> longest_palindromic_subsequence("55055901565109",14)
+    9
+    """
+
+    # creating an array to store the values generated
+    dp_table = [[1 for i in range(length)] for i in range(length)]
```

Comment:
Can we come up with a more self-documenting name than `dp_table`?

Reply:
Would array be fine? If not what would you suggest?

Reply:
What would we call the value that is in each cell of this table?  Are the _weights_ or _probabilities_ or _angstroms_ or what?